pkg/kubelet/cm/memorymanager/policy_static.go

```diff
@@ -109,7 +154,7 @@ func (p *staticPolicy) Allocate(s state.State, pod *v1.Pod, container *v1.Contai
 	// topology manager returned the hint with NUMA affinity nil
 	// we should use the default NUMA affinity calculated the same way as for the topology manager
 	if hint.NUMANodeAffinity == nil {
-		defaultHint, err := p.getDefaultHint(s, requestedResources)
+		defaultHint, err := p.getDefaultHint(s, requestedResources, string(pod.UID))
```

Comment:
Strange change, I haven't done that. It must have lasted refactoring. Will drop it.